binding/binding-mri.cpp

```diff
@@ -779,9 +779,50 @@ static bool processReset(bool rubyExc) {
 	return 0;
 }
 
+#if RAPI_FULL > 187
+static VALUE newStringUTF8(const char *string, long length) {
+    return rb_enc_str_new(string, length, rb_utf8_encoding());
+}
+#else
+#define newStringUTF8 rb_str_new
+#endif
+
+struct evalArg {
+    VALUE string;
+    VALUE filename;
+};
+
+static VALUE evalHelper(evalArg *arg) {
+    VALUE argv[] = {arg->string, Qnil, arg->filename};
+    return rb_funcall2(Qnil, rb_intern("eval"), ARRAY_SIZE(argv), argv);
+}
+
+static VALUE evalString(VALUE string, VALUE filename, int *state) {
+    evalArg arg = {string, filename};
+    return rb_protect((VALUE(*)(VALUE))evalHelper, (VALUE)&arg, state);
+}
+
+static void runCustomScript(const std::string &filename) {
+    std::string scriptData;
+    
+    if (!readFileSDL(filename.c_str(), scriptData)) {
+        showMsg(std::string("Unable to open '") + filename + "'");
+        return;
+    }
+    
+    evalString(newStringUTF8(scriptData.c_str(), scriptData.size()),
+               newStringUTF8(filename.c_str(), filename.size()), NULL);
+}
+
 RB_METHOD_GUARD(mriRgssMain) {
     RB_UNUSED_PARAM;
-    
+
+    /* Execute postload scripts */
+    const Config &conf = shState->rtData().config;
+    for (std::vector<std::string>::const_iterator i = conf.postloadScripts.begin();
+         i != conf.postloadScripts.end(); ++i)
+        runCustomScript(*i);
```

Comment:
Please enclose the body of this `for` loop in braces, so that we're at lower risk of introducing a gotofail bug in the future.

Reply:
Yes, originally I had just copied preloadScripts, it looks like Wayward updated it after I made this PR on [#9e8132ba2504eed0d1c2fda2f376cbdcf57de68a](https://github.com/mkxp-z/mkxp-z/commit/9e8132ba2504eed0d1c2fda2f376cbdcf57de68a)